src/Resources/skeleton/crud/templates/new.tpl.php

```diff
@@ -0,0 +1,9 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    <h1>Create new <?= $entity_class_name; ?></h1>
+    {{ form_start(form) }}
+    {{ form_widget(form) }}
```

Comment:
<?php foreach ($entity_fields as $field): ?>{{ form_row(form.<?= ucfirst($field['fieldName']); ?>) }}
    <?php endforeach; ?>

would be my preference here, and I would use a seperate `_form.html.twig` to generate the same form for the two occurrences. https://github.com/symfony/demo/blob/master/templates/admin/blog/_form.html.twig

Reply:
do we really need to generate whole form with all fields separated?